templates/email/html/order_successful_self_service.php

```diff
@@ -52,20 +52,16 @@
     <tbody>
 
 
-        <tr><td style="padding-top:20px;">
-
-
-        <?php if (Configure::read('app.sendTaxViaMail')) { ?>
-
+        <?php if (Configure::read('app.showTaxInOrderConfirmationEmail')) { ?>
 
```

Comment:
here - into this bracket, then the next message will not have a padding to the table before, which leaves it sticking together, which does not look nice in the email.

Reply:
sorry, it was a bit hard to read and I thought it was an error.
should be ok now:
https://github.com/foodcoopshop/foodcoopshop/blob/develop/templates/email/html/order_successful_self_service.php